content/en/docs/reference/networking/service-protocols.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Protocols for Services
+content_type: reference
+---
+
+<!-- overview -->
+If you configure a {{< glossary_tooltip text="Service" term_id="service" >}},
+you can select from any network protocol that Kubernetes supports.
+
+Kubernetes supports the following protocols with Services:
+
+- [`SCTP`](#protocol-sctp)
+- [`TCP`](#protocol-tcp) _(the default)_
+- [`UDP`](#protocol-udp)
+
+When you define a Service, you can also specify the
+[application protocol](/docs/concepts/services-networking/service/#application-protocol)
+that it uses.
+
+This document details some special cases, all of them typically using TCP
+as a transport protocol:
+
+- [HTTP](#protocol-http-special) and [HTTPS](#protocol-http-special)
+- [PROXY protocol](#protocol-proxy-special)
+- [TLS](#protocol-tls-special) termination at the load balancer
+
+<!-- body -->
+## Supported protocols {#protocol-support}
+
+There are 3 valid values for the `protocol` of a port for a Service:
+
+### `SCTP` {#protocol-sctp}
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+When using a network plugin that supports SCTP traffic, you can use SCTP for
+most Services. For `type: LoadBalancer` Services, SCTP support depends on the cloud
+provider offering this facility. (Most do not).
+
+SCTP is not supported on nodes that run Windows.
+
+#### Support for multihomed SCTP associations {#caveat-sctp-multihomed}
+
+The support of multihomed SCTP associations requires that the CNI plugin can support the assignment of multiple interfaces and IP addresses to a Pod.
+
+NAT for multihomed SCTP associations requires special logic in the corresponding kernel modules.
+
+{{< note >}}
+The kube-proxy does not support the management of SCTP associations when it is in userspace mode.
+{{< /note >}}
```

Comment:
userspace kube-proxy is going to be removed , I think we started deprecation some releases ago
https://github.com/kubernetes/kubernetes/issues/103860

Reply:
duplicate, we already discussed it here https://github.com/kubernetes/website/pull/30817/files#r830832828

Reply:
That's OK. When we remove it, we should remove this note.